<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SvrOptDialog</class>
 <widget class="QWidget" name="SvrOptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="12" column="2">
    <widget class="QDoubleSpinBox" name="AntOff1">
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="3">
    <widget class="QLabel" name="Label7">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Output Log Level</string>
     </property>
    </widget>
   </item>
   <item row="20" column="7">
    <widget class="QPushButton" name="BtnLocalDir">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="Label12">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Antenna Info</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="Label13">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Receiver Info</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Log File</string>
     </property>
    </widget>
   </item>
   <item row="18" column="2" colspan="5">
    <widget class="QLineEdit" name="SrcTblFileF"/>
   </item>
   <item row="18" column="7">
    <widget class="QPushButton" name="BtnSrcTblFile">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="17" column="2" colspan="5">
    <widget class="QLineEdit" name="LogFileF"/>
   </item>
   <item row="17" column="7">
    <widget class="QPushButton" name="BtnLogFile">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="3">
    <widget class="QLabel" name="Label9">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>File Swap Margin</string>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <widget class="QLabel" name="Label10">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>HTTP/NTRIP Proxy</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="Label2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Server Cycle </string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="SvrBuffSize">
     <property name="suffix">
      <string> bytes</string>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="value">
      <number>16384</number>
     </property>
    </widget>
   </item>
   <item row="15" column="2" colspan="5">
    <widget class="QComboBox" name="RelayMsg">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(1) -&gt; (0)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(2) -&gt; (0)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(3) -&gt; (0)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QDoubleSpinBox" name="AntPos1">
     <property name="suffix">
      <string>Â°</string>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="DataTimeout">
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="value">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="12" column="5" colspan="2">
    <widget class="QDoubleSpinBox" name="AntOff3">
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="20" column="2" colspan="5">
    <widget class="QLineEdit" name="LocalDir">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="6" colspan="2">
    <widget class="QComboBox" name="TraceLevelS">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Level 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Level 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Level 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Level 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Level 5</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="5" colspan="2">
    <widget class="QDoubleSpinBox" name="AntPos3">
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="7">
    <widget class="QPushButton" name="BtnPos">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="Label8">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Lat/Lon/Height</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="3">
    <widget class="QCheckBox" name="NmeaReqT">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>NMEA Cycle</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="3" colspan="2">
    <widget class="QDoubleSpinBox" name="AntPos2">
     <property name="suffix">
      <string>Â°</string>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="20" column="0">
    <widget class="QLabel" name="Label4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>FTP/HTTP Local Dir</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="StaInfoSel">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Station ID</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="3">
    <widget class="QLabel" name="Label6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Period of Rate</string>
     </property>
    </widget>
   </item>
   <item row="22" column="6">
    <widget class="QPushButton" name="BtnOk">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;OK</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Relay Messages</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2" colspan="6">
    <widget class="QLineEdit" name="RcvInfo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>ReceiverInfo</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="Label11">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Offset E/N/U</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QSpinBox" name="StationId">
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>1234</number>
     </property>
    </widget>
   </item>
   <item row="22" column="7">
    <widget class="QPushButton" name="BtnCancel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="SvrCycle">
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="6" column="6" colspan="2">
    <widget class="QSpinBox" name="NmeaCycle">
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>NTRIP Source Table</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6" colspan="2">
    <widget class="QSpinBox" name="FileSwapMarginE">
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSpinBox" name="ConnectInterval">
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>2000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="Label1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Buffer Size</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="Label5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Reconnect Interval</string>
     </property>
    </widget>
   </item>
   <item row="21" column="2" colspan="6">
    <widget class="QLineEdit" name="ProxyAddr">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="6" colspan="2">
    <widget class="QSpinBox" name="AvePeriodRate">
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="13" column="2" colspan="6">
    <widget class="QLineEdit" name="AntInfo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>AntInfo</string>
     </property>
    </widget>
   </item>
   <item row="12" column="3" colspan="2">
    <widget class="QDoubleSpinBox" name="AntOff2">
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="Label3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Inactive Timeout</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Progress Bar Range</string>
     </property>
    </widget>
   </item>
   <item row="16" column="2" colspan="5">
    <widget class="QSpinBox" name="ProgBarR">
     <property name="suffix">
      <string> kb</string>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>2000</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>BtnOk</tabstop>
  <tabstop>BtnCancel</tabstop>
  <tabstop>SvrBuffSize</tabstop>
  <tabstop>AvePeriodRate</tabstop>
  <tabstop>SvrCycle</tabstop>
  <tabstop>FileSwapMarginE</tabstop>
  <tabstop>DataTimeout</tabstop>
  <tabstop>TraceLevelS</tabstop>
  <tabstop>ConnectInterval</tabstop>
  <tabstop>NmeaReqT</tabstop>
  <tabstop>NmeaCycle</tabstop>
  <tabstop>StaInfoSel</tabstop>
  <tabstop>StationId</tabstop>
  <tabstop>AntPos1</tabstop>
  <tabstop>AntPos2</tabstop>
  <tabstop>AntPos3</tabstop>
  <tabstop>BtnPos</tabstop>
  <tabstop>AntOff1</tabstop>
  <tabstop>AntOff2</tabstop>
  <tabstop>AntOff3</tabstop>
  <tabstop>AntInfo</tabstop>
  <tabstop>RcvInfo</tabstop>
  <tabstop>RelayMsg</tabstop>
  <tabstop>ProgBarR</tabstop>
  <tabstop>LogFileF</tabstop>
  <tabstop>BtnLogFile</tabstop>
  <tabstop>SrcTblFileF</tabstop>
  <tabstop>BtnSrcTblFile</tabstop>
  <tabstop>LocalDir</tabstop>
  <tabstop>BtnLocalDir</tabstop>
  <tabstop>ProxyAddr</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
